conditions/conditions.go

```diff
@@ -84,7 +84,7 @@ func (c *condition) Get(ctx context.Context) (*metav1.Condition, error) {
 
 // Set implements conditions.Set
 func (c *condition) Set(ctx context.Context, status metav1.ConditionStatus, option ...Option) error {
-	operatorCond := &apiv1.OperatorCondition{}
+	operatorCond := &apiv2.OperatorCondition{}
 	err := c.client.Get(ctx, c.namespacedName, operatorCond)
 	if err != nil {
 		return ErrNoOperatorCondition
```

Comment:
Please consider returning the original error, or at least wrap it.

Reply:
I support this change, but recommend doing so in a separate commit.

Reply:
See https://github.com/operator-framework/operator-lib/pull/65